airbyte-integrations/connectors/source-bing-ads/source_bing_ads/report_streams.py

```diff
@@ -154,7 +154,7 @@ def request_params(
     def get_start_date(self, stream_state: Mapping[str, Any] = None, account_id: str = None):
         if stream_state and account_id:
             if stream_state.get(account_id, {}).get(self.cursor_field):
-                return pendulum.parse(self.get_report_record_timestamp(stream_state[account_id][self.cursor_field]))
+                return pendulum.parse(stream_state[account_id][self.cursor_field])
```

Comment:
I would expect a test to fail if we change how we use the state. Should we add a test to document this?

Reply:
tests added